spec/services/service_provider_updater_spec.rb

```diff
@@ -10,7 +10,8 @@
     [
       {
         id: 'big number',
-        updated_at: '2010-01-01 00:00:00',
+        created_at: '2010-01-01 00:00:00'.to_datetime,
+        updated_at: '2010-01-01 00:00:00'.to_datetime,
```

Comment:
does JSON.parse automatically convert those to `DateTime` objects? Is there magic here I don't see?